extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,256 @@
+# 3DTILES_metadata Extension
```

Comment:
Put the version number in this top level section instead of in the Status section, to be consistent with the main 3D Tiles specification. Something like:

**Version 0.0.0** November 6, 2020

Reply:
I added a little more format to this:

```
Version: **Version 0.0.0**
Revision Date: November 6, 2020.
```